inspirehep/modules/orcid/builder.py

```diff
@@ -0,0 +1,277 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2014-2017 CERN.
+#
+# INSPIRE is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this license, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+"""
+Builds an ORCID work record.
+"""
+
+from __future__ import absolute_import, division, print_function
+
+
+class OrcidBuilder(object):
```

Comment:
if what ORCID wants is XML, maybe the builder should build an XML document instead? you could use `lxml.etree` or maybe the [E-factory](http://lxml.de/tutorial.html#the-e-factory), whatever you think is nicer.

Reply:
They do support both, it's just they only publish schemas for XML. I will look into these, it would be more straightforward to validate XML.